lib/ci-helper.ts

```diff
@@ -562,6 +578,10 @@ export class CIHelper {
                         argument} already allowed to use GitGitGadget.`);
                 }
             } else if (command === "/disallow") {
+                if (!argument) {
+                    argument = pr.author;
+                }
+
```

Comment:
Again, for readability, I would prefer `const accountName = argument || await getPRAuthor();`.

What do you think?